kernel/src/main/scala/cats/kernel/Priority.scala

```diff
@@ -0,0 +1,65 @@
+package cats.kernel
```

Comment:
This doesn't appear to be used anywhere in cats. Is there a particular reason to include it in cats-kernel?